src/ode_solver/relaxation_runge_kutta/empty_RRK_base.h

```diff
@@ -0,0 +1,56 @@
+#ifndef __EMPTY_RRK_BASE_H__
+#define __EMPTY_RRK_BASE_H__
+
+#include "dg/dg_base.hpp"
+#include "ode_solver/runge_kutta_methods/rk_tableau_base.h"
+
+namespace PHiLiP {
+namespace ODE {
+
+/// Empty class stored by RK solvers which do not use RRK, and also do not need to calculate numerical entropy.
+/// Functions in this class are empty or return a "dummy" value. 
```

Comment:
can they not just hold a `nullptr` by default? then you could make this a purely virtual class

Reply:
Ahh I see, yeah this works